src/services/completions.ts

```diff
@@ -1346,7 +1350,13 @@ function createCompletionEntry(
         }
     }
 
-    if (origin?.kind === SymbolOriginInfoKind.TypeOnlyAlias) {
+    if ((origin?.kind === SymbolOriginInfoKind.TypeOnlyAlias)) {
+        hasAction = true;
+    }
+
+    if ((contextToken && isPropertyAssignment(contextToken.parent) && findNextToken(contextToken, contextToken?.parent, sourceFile)?.kind !== SyntaxKind.CommaToken &&
+        completionKind === CompletionKind.ObjectPropertyDeclaration)) {

```

Comment:
`completionKind === CompletionKind.ObjectPropertyDeclaration` is the cheapest of these conditions to check (along with `contextToken`), so it should be moved before the more expensive ones (particularly the `findNextToken` call) so the harder work can be short-circuited.

(Also, it looks like there’s an extra pair of parens around this whole expression.)